src/focus_manager/txm_platform.js

```diff
@@ -504,11 +510,16 @@ export class TXMPlatform {
                 self.model = details[1];
                 self.version = details[2];
             } else {
-                var detailSubstring = userAgent.substring(userAgent.indexOf("(") + 1, userAgent.indexOf(")"));
-                var detailSplit = detailSubstring.split(";");
-                self.model = detailSplit[2].split(" ").find(part => {return part});
-                const versionParts = detailSplit[1].split(" ");
-                self.version = versionParts[versionParts.length - 1];
+                var groupStart = userAgent.indexOf("(");
+                if (groupStart >= 0) {
```

Comment:
What if this condition is false? is `self.model` and `self.version` not being defined a problem? Sorry -- can't dive too deep right now

Reply:
there is a default value of "Unknown" for them

Reply:
the .model assignment part is technically a bug, but is overridden by the callers to do something better for the actual platform

Reply:
so, let me know if you want a fix and a version bump

Reply:
nah, just wanted to check if something was forgotten